modules/configure-slack.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-slack.adoc
+:_module-type: PROCEDURE
+[id="configure-slack_{context}"]
+= Configuring Slack
+
+Start by creating a new Slack App, and get the webhook URL.
+
+.Procedure
+. Create a new Slack App:
++
+[NOTE]
+====
+If you want to use an existing Slack App, go to https://api.slack.com/apps and select an app.
+====
+.. Navigate to https://api.slack.com/apps/new.
+.. Enter the *App Name* and choose a *Development Slack Workspace* to install your app.
+.. Click *Create App*.
+. On the settings page, *Basic Information* section, select *Incoming Webhooks* (under *Add features and functionality*).
```

Comment:
```suggestion
. Under *Basic Information*, click *Incoming Webhooks*.
```
(I'd avoid parentheses and also try to minimize describing the Slack interface, which is subject to change at any time.)